docs/UserGuide.md

```diff
@@ -469,39 +483,35 @@ Volunteers who are already assigned to the specified event, not available on the
 
 <div style="page-break-after: always;"></div>
 
-## Saving the data
+## About Saving Your Work
 
-VolunSync's data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+Don't worry about saving - VolunSync automatically saves everything you do! Every change you make is safely stored on your computer.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## FAQ
 
 **Q**: Can I undo an action that I just made?<br>
-**A**: Currently VolunSync does not support the `undo` function, but we are planning to implement it in the future so stay tuned for the update!
+**A**: Not yet, but we're working on adding an `undo` feature in our next update. Stay tuned!
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Known Issues
 
-1. **Multi-Screen Setup Issue**
-   **Problem:** If you move the application to a secondary screen and later switch to using only the primary screen, the GUI may open off-screen when you restart the app.
-   **Solution:**
-    - To fix this, locate and delete the `preferences.json` file, which stores the app's window position.
-    - After deleting the file, restart the application, and the GUI will open correctly on the primary screen.
-
-1. **Help Window Remaining Minimized**
-   **Problem:** If you minimize the Help Window and run the `help` command (or use the `Help` menu or press the `F1` key), the Help Window will remain minimized instead of reopening.
-   **Solution:**
-    - Manually restore the Help Window from your taskbar or dock.
-    - Alternatively, close the minimized window and run the `help` command again to open a new Help Window.
+1. **Using Multiple Screens?**
+   If you move VolunSync to a second screen and later disconnect it:
+    - The program might open off-screen next time
+    - To fix this, just find and delete the `preferences.json` file
+    - Start VolunSync again, and it'll open on your main screen
 
```

Comment:
Add the dont touch data json manually here?

Reply:
added it in